src/lib/y2network/sequences/interface.rb

```diff
@@ -2,7 +2,11 @@
 
 module Y2Network
   module Sequences
-    # Sequence for interface
+    # The responsibility of this class is to drive workflow for sequence of dialogs.
+    #
+    # In this case for interface allowed operations are add for adding new interface,
```

Comment:
Would modify it a little be in order to be more readable, at least the first paragraph was not clear in my first read attempt.

What about something like:

```In this case, allowed operations for interface are, 'add' for adding a new interface, 'edit' for editing an existing device / interface or 'init_s390' for s390 initialization which needs specific activation before using it...```